ui/src/store/modules/auth.ts

```diff
@@ -14,14 +14,15 @@ const cookieStorage = new CookieStorage({
   sameSite: 'Strict',
 })
 
-const oidcSettings: VuexOidcClientSettings = {
+const oidcSettings = {
   redirectUri: new URL('oidc-callback', base).toString(),
   postLogoutRedirectUri: new URL('logout', base).toString(),
   responseType: 'code',
   automaticSilentRenew: true,
   stateStore: new WebStorageStateStore({ store: cookieStorage }),
+  monitorSession: false,
   ...oidc,
-}
+} as VuexOidcClientSettings
```

Comment:
This is necessary because of bad typing in `vuex-oidc`. `VuexOidcClientSettings` extends from `OidcClientSettings`, while it should extend from `UserManagerSettings`.